wlauto/common/linux/device.py

```diff
@@ -318,13 +318,13 @@ def is_installed(self, name):  # pylint: disable=unused-argument,no-self-use
 
             please see: https://pythonhosted.org/wlauto/writing_extensions.html""")
 
-    def is_network_connected(self, ip_address='8.8.8.8'):
+    def is_network_connected(self, ip_address='8.8.8.8'):  # pylint: disable=arguments-differ
```

Comment:
Updated. Removed the argument for the interface method. Device subclasses can have their own internal implementation to check for internet e.g. `LinuxDevice` uses `_ping_server()`